lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -632,6 +635,10 @@ def add_normalized_threeds_2_data(xml, payment_method, options)
         xml.tag!('paresStatus', threeds_2_options[:authentication_response_status]) if threeds_2_options[:authentication_response_status]
       end
 
+      def infer_commerce_indicator?(options, cc_brand)
+        options[:commerce_indicator].blank? && ECI_BRAND_MAPPING[cc_brand].present?
```

Comment:
Asked cybersource what are the commerce indicators for the supported brands absent from `ECI_BRAND_MAPPING`. If a commerce indicator could be inferred for all the supported brands, this code could be simplified.